triton-vm/Cargo.toml

```diff
@@ -27,7 +27,7 @@ default-features = false
 
 [dependencies]
 twenty-first = { version = "0.7.2" }
-triton-profiler = { version = "0.7.2" }
+triton-profiler = { version = "0.7.2", path = "../triton-profiler" }
```

Comment:
We probably want users to use the crates.io package, not from the local path.

Reply:
Correct. I'll change that now upon release; we needed to use the local one while developing.